src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+// controllers
+const ordersController = require('../controllers/orders.controller');
+/**
+ * @swagger
+ * /orders/{id}:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get orders by order id
+ *    description:
+ *      Will return all orders for specific id.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: id
+ *       schema:
+ *         type: integer
+ *         required: true
```

Comment:
looks good but when we don't write any integer as id,it should not execute the request.Can refer favorites and users router files